Automattic/wp-calypso
# Add new package: @automattic/comment-block-editor

## Proposed Changes

This adds a new package to be consumed from Calypso and WPCOM. This is a simple gutenberg editor built specifically for Verbum Comments. View the original source of the code [here](https://github.com/Automattic/verbum/tree/trunk/src/editor). This is just adding the package to calypso and is not being used yet.

The build for this uses Vite which creates its own `node_modules` inside the package.

## Testing Instructions

Smoke test. Since this is not actually being wired up into anything yet there is no point to test. It may be good to check and make sure nothing has broken with the build, install, or start process.

## Pre-merge Checklist
